.changeset/shiny-chefs-invent.md

```diff
@@ -0,0 +1,5 @@
+---
+"@hopper-ui/components": patch
```

Comment:
ugh normally it would be. idk anymore.. I thought I remember that since we are not actually released, we would just to patches so that we could test in alpha, but I'm not sure..

Reply:
but at one point I did do a minor, forgetting about that.

Reply:
Yes let's not worry before batch 1.